### PR TITLE
Feature/risk analysis data

### PR DESCRIPTION
**Pull request by Gatchang-nyawargak**
## What this PR does?
Added  risk analysis report

## Type of change

Specify the kind of PR/MR

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Optimization
- [ ] Improvement
- [ ] Bug fix

## How can it be tested?
- Please provide step-by-step guide on how to test
- If web or mobile, screenshots will be very helpful in addition

# Where is it documented?
- List JIRA tickets
[Risk Analysis Report](https://cherotichdeborahb.atlassian.net/jira/software/projects/SCRUM/boards/1?selectedIssue=SCRUM-189)

### CODE REVIEW

**Comment by Gatchang-nyawargak** [comment]
Screenshot from 2024-09-22 13-31-34
Screenshot from 2024-09-22 13-31-24

**Review by majidhawa — DISMISSED** [review]

**Review by Nancy-Wainaina — APPROVED** [review]

**Review by Deborah-Cherotich — APPROVED** [review]

**Review by Susan-Favour — DISMISSED** [review]

File: investika-dashboard/src/app/risk-analysis-report/page.tsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+import Layout from '../component/Layout';
+
+const AchievementManagement: React.FC = () => {
+  const { achievements, loading: loadingAchievements, error: errorAchievements } = useAchievements();
+  const { users, loading: loadingUsers, error: errorUsers } = useUsers();
+  const { simulations, loading: loadingSimulations, error: errorSimulations } = useSimulations();
+  const [currentPage, setCurrentPage] = useState(1);
+  const achievementsPerPage = 10;
+
+  useEffect(() => {
```
Comment (hunterachieng) [inline investika-dashboard/src/app/risk-analysis-report/page.tsx:16]: delete this

File: investika-dashboard/src/app/user-tracking/page.tsx [hunk 2]
```diff
@@ -1,38 +1,30 @@
...
   const { virtualMoney, loading: loadingMoney, error: errorMoney } = useVirtualMoney();
-  const { users, loading: loadingUsers, error: errorUsers } = useUsers(); 
+  const { users, loading: loadingUsers, error: errorUsers } = useUsers();
   const { simulations, loading: loadingSimulations, error: errorSimulations } = useSimulations();
-
   const [currentPage, setCurrentPage] = useState(1);
-  const usersPerPage = 10; 
-
+  const usersPerPage = 10;
   useEffect(() => {
```
Comment (hunterachieng) [inline investika-dashboard/src/app/user-tracking/page.tsx:13]: Remove this as well

File: investika-dashboard/utils/fetchAchievement.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  reward_type: string;
+  date_achieved: string;
+}
+
+export const fetchAchievement = async (): Promise<Achievements[]> => {
+  try {
+    const response = await fetch('/api/achievements');
+
+    // Log the entire response for debugging
+    console.log("API Response: ", response);
```
Comment (hunterachieng) [inline investika-dashboard/utils/fetchAchievement.ts:13]: Delete this

**Review by Susan-Favour — APPROVED** [review]

**Review by majidhawa — APPROVED** [review]

**Review by Deborah-Cherotich — APPROVED** [review]

**Review by Nancy-Wainaina — APPROVED** [review]

**Review by Deborah-Cherotich — APPROVED** [review]

**Review by hunterachieng — APPROVED** [review]

**Pull request merged**